src/networking.c

```diff
@@ -2915,6 +3020,8 @@ void clientCommand(client *c) {
 "    Return the client ID we are redirecting to when tracking is enabled.",
 "GETNAME",
 "    Return the name of the current connection.",
+"GETMETADATA [<field>]",
```

Comment:
I'm not sure if this feature is useful, but I have no objection, since it is harmless.

But I don't think metadata is a good name, it looks like a server's side data, but redis server does not use it, it's just user defined data, maybe "custom_data" is better.

Reply:
it sounds good to me